spacy_loggers/tests/test_lookup.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import re
+
+from spacy_loggers.util import matcher_for_regex_patterns
+from .util import load_logger_from_config
+
+
+valid_config_string = """
+[nlp]
+lang = "en"
+pipeline = ["tok2vec"]
+
+[components]
+
+[components.tok2vec]
+factory = "tok2vec"
+
+[training]
+
+[training.logger]
+@loggers = "spacy.LookupLogger.v1"
+patterns = ["^(p|P)ytorch", "zeppelin" ]
```

Comment:
And one more remnant of the capturing group ;-)

Reply:
Exterminated.